packages/styled-components/src/models/StyleTags.js

```diff
@@ -58,7 +58,11 @@ const addUpUntilIndex = (sizes: number[], index: number): number => {
 
 /* create a new style tag after lastEl */
 const makeStyleTag = (target: ?HTMLElement, tagEl: ?Node, insertBefore: ?boolean) => {
-  const el = document.createElement('style');
+  let targetDocument = document;
+  if(target) targetDocument = target.ownerDocument;
+  else if(tagEl) targetDocument = tagEl.ownerDocument;
```

Comment:
I'd write this as:

`const targetDocument = target ? target.ownerDocument : tagEl ? tagEl.ownerDocument : document`

but that's just my anti-mutability bias 🤷‍♀

Reply:
@Jessidhia thank you for reviewing! I agree with anti-mutability, though, the eslint rule (no-nested-ternary) didn't let me do so :)